resources/lang/hu/faults.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+    "faults" => "Hibák",
+    "created_at" => "Időpont",
+    "location" => "Helyszín",
+    "description" => "Leírás",
+    "status" => "Állapot",
+    "submit" => "Küldés",
+    "new_fault" => "Hiba bejelentése",
+    "cancel" => "Mégsem",
+    "confirm_message" => "Biztosan meg akarja változtatni a hiba állapotát a következőre: ",
+    "confirm" => "Megerősítés",
+    "done" => "Kész",
+    "wont_fix" => "Nem megoldandó",
+    "unseen" => "Nem látott",
+    "seen" => "Látott",
```

Comment:
megszemlélve?

Reply:
That's possible, but doesn't that sound too archaic?

Reply:
Yes, that's what Facebook also uses. I avoided that because I find that more appropriate in a 1-1 conversation, but I guess you're right, it's still better. Have changed that.